src/main/java/org/wso2/carbon/connector/pcml/PCMLPool.java

```diff
@@ -0,0 +1,64 @@
+package org.wso2.carbon.connector.pcml;
+
+import com.ibm.as400.access.AS400ConnectionPool;
+
+public class PCMLPool implements PCMLPoolMBean {
+
+    AS400ConnectionPool as400ConnectionPool;
+    int maxConnections;
+    long maxInactivity;
+    int maxLifetime;
+    int maxUseCount;
+    int maxUseTime;
+    boolean isRunMaintenance;
+    boolean isThreadUsed;
+    int cleanupInterval;
+    boolean pretestConnections;
+
+
+    public PCMLPool(AS400ConnectionPool connectionPool) {
```

Comment:
Do we have any method in the `AS400ConnectionPool` to get the currently active connections?

Reply:
searched but did not find it

Reply:
Shall we check [getActiveConnectionCount](https://www.ibm.com/docs/api/v1/content/ssw_ibm_i_74/rzahh/javadoc/com/ibm/as400/access/AS400ConnectionPool.html#getActiveConnectionCount(java.lang.String,%20java.lang.String))(java.lang.String systemName, java.lang.String userID) and [getAvailableConnectionCount](https://www.ibm.com/docs/api/v1/content/ssw_ibm_i_74/rzahh/javadoc/com/ibm/as400/access/AS400ConnectionPool.html#getAvailableConnectionCount(java.lang.String,%20java.lang.String))(java.lang.String systemName, java.lang.String userID) ?

Reply:
@markelari, seems like you have missed pushing the changes